src/app/product-detail/product-detail.component.scss

```diff
@@ -0,0 +1,68 @@
+// .product-detail {
+//   margin: 20px;
+// }
+
+// .product-detail h2 {
+//   font-size: 24px;
+//   margin-bottom: 10px;
+// }
+
+// .product-detail p {
+//   font-size: 18px;
+//   margin: 5px 0;
+// }
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.products {
+  padding-bottom: 10px;
+}
+
+.remove-s {
+  padding-left: 110px;
+  display: flex;
+  flex-flow: row nowrap;
+  flex: 1;
+  justify-content: space-evenly;
+}
+
+.small-image {
+  padding-right: 20px;
+}
```

Comment:
replace this rule with:

::ng-deep p-image img {
    max-width: 50px;
    height: 100%;
    object-fit: cover;
    aspect-ratio: 1;
}

Reply:
i did face an error when i commited in the (::ng-deep p-image img), it says that it is an unkown pseudo element.